cypress/integration/Project.spec.js

```diff
@@ -33,4 +33,25 @@ describe('Project', () => {
   it('has a picture for every project', () => {
     cy.get(`img[src*="/"]`).should('have.length.above', 100)
   })
+  it('shows all tags', () => {
+    cy.get('.TagCluster__TagList-x84l3-0 > li').should('have.length', 12)
```

Comment:
These classnames look like they are automatically generated. Will the test still work, if the names change?

Reply:
I wondered about the same thing, apparently styled-components always uses the same class names, but if someone knows a better selector, I'd change it.